akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpMessage.scala

```diff
@@ -284,6 +302,10 @@ final class HttpRequest(
   override def isRequest = true
   override def isResponse = false
 
+  @deprecated("for backwards compatibility", "10.2.0")
```

Comment:
```suggestion
  @deprecated("use constructor with `attributes` instead", "10.2.0")
```
or should it even be `private[akka]`?